### ISSUE TITLE
What is a Logit? // ML dictionary / 

### ISSUE THREAD

**Issue by joeyklee** [shiffman/Intro-ML-Arts-IMA#28]
https://stackoverflow.com/questions/41455101/what-is-the-meaning-of-the-word-logits-in-tensorflow

More of a meta discussion: maybe it is good to have a ML term dictionary with creative examples and metaphors? 

e.g. 
* http://18f.github.io/API-All-the-X/pages/what_are_APIs-anecdotes_and_metaphors/
* http://metaphorical.ly/
* https://sidewaysdictionary.com/#/ via @auremoser

**Comment by ellennickles** [comment]
Yes! This came up in conversation yesterday in the context of a possible resource for the ml5 site. 

Some technical references sans the types of creative examples suggested above:
- [Google's ML Glossary](https://developers.google.com/machine-learning/glossary/)
- [NVIDIA's Deep Learning Glossary](https://www.nvidia.com/content/g/pdfs/nvidia-deeplearning-glossary-llkcmb.pdf?ncid=em-nur-47510&mkt_tok=eyJpIjoiTjJSa016RTFNbU0zWW1ZMyIsInQiOiJnamp6MUlDVVloQjNqaWFFcjZJeVorclh5WEg4c1ZsaWREbzBwYVwvSG5tVUVLVjNmSFpYcEVTNWt2VzFkTWpBWmVjbXc1OXl3cnF6SjNUNU5abmNrM0J1TGtUQWlhN3gwOW9YRlUxNHI2VDJMV0p5QmZxQmpsekxIakRIT0lSbFEifQ%3D%3D)
